src/promptflow/tests/sdk_cli_azure_test/e2etests/test_flow_operations.py

```diff
@@ -15,66 +16,30 @@
 data_dir = tests_root_dir / "test_configs/datas"
 
 
-@pytest.fixture
-def client(remote_client):
-    return remote_client
-
-
 # TODO: enable the following tests after test CI can access test workspace
 @pytest.mark.timeout(timeout=DEFAULT_TEST_TIMEOUT, method=PYTEST_TIMEOUT_METHOD)
 @pytest.mark.e2etest
-@pytest.mark.skip(reason="This test is not ready yet.")
+@pytest.mark.usefixtures(
+    "mock_set_headers_with_user_aml_token",
+    "single_worker_thread_pool",
+    "vcr_recording",
+)
+@pytest.mark.skip(reason="Enable this after recording is ready for flow operations.")
```

Comment:
Will these test run in live mode?

Reply:
I think so, as soon as the recording is ready. All of those tests should be run in live mode regularly